src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -15,18 +16,37 @@ import { createFragmentContainer, graphql } from "react-relay"
 import { data as sd } from "sharify"
 
 import { ArtworkSidebarBidAction_artwork } from "__generated__/ArtworkSidebarBidAction_artwork.graphql"
+import { ArtworkSidebarBidAction_me } from "__generated__/ArtworkSidebarBidAction_me.graphql"
 import { SystemContextConsumer } from "Artsy"
 import * as Schema from "Artsy/Analytics/Schema"
 import track from "react-tracking"
 
 export interface ArtworkSidebarBidActionProps {
   artwork: ArtworkSidebarBidAction_artwork
+  me: ArtworkSidebarBidAction_me
 }
 
 export interface ArtworkSidebarBidActionState {
   selectedMaxBidCents?: number
 }
 
+const RegisterToBidButton = (props: { onClick: () => void }) => {
```

Comment:
```
RegisterToBidButton: React.FC<{ onClick: ... }>
```

Reply:
Will do – but what's the brief "why" here? More performant, just better types?

Reply:
Types / Intellisense / "the react way"